android/src/main/java/com/apparence/camerawesome/CameraSetup.java

```diff
@@ -104,6 +112,25 @@ public void onOrientationChanged(int i) {
         orientationEventListener.enable();
     }
 
+    /**
+     * Returns the natural orientation of the device: Configuration.ORIENTATION_LANDSCAPE or
+     * Configuration.ORIENTATION_PORTRAIT.
+     * The result should be consistent no matter the orientation of the device
+     */
+    public static int getScreenNaturalOrientation(@NonNull final Context context) {
```

Comment:
The original name of this method sounds a bit better than getScreenNaturalOrientation for me.

```suggestion
    public static int getDeviceDefaultOrientation(@NonNull final Context context) {
```

Reply:
The reason I used natural is because that term is also used at https://developer.android.com/training/camerax/orientation-rotation. I'll change "Screen" to "Device" and leave the natural part in if you're fine with that.

Reply:
Okay, accepted 👍